mxd/modules/connector/variables.tf

```diff
@@ -62,4 +62,21 @@ variable "ssi-config" {
     oauth-clientsecret = ""
     oauth-secretalias  = ""
   }
-}
\ No newline at end of file
+}
+
+variable "azure-account-name" {
+  description = "Azure Account Name for the connector"
+}
+
+variable "azure-account-key" {
+  description = "Azure Account Key for the connector"
```

Comment:
could provide a default value, eg. a random generated one

Reply:
We do have a default value for both Alice and Bob in top level variables.tf.
We have some limitations on using a random generated password.
Azure temp keys (Azure SAS) has been generated using the default account name / account key and added as static value in the code.
We can generate Azure SAS dynamically, using Azure CLI pod, but we can't share this key to the connector pod as a variable in vault postStart script (kind of Terraform limitation, I asked about it in reddit and they suggested to use ArgoCD, ArgoCD does support workflows where output of one pod can be shared with next pod in the workflow). Good thing is this SAS key doesn't change if parameters are same (Account Name / key / permissions etc), that's why I pre-generated it and added as static value.
Also I have added steps how to generate this SAS key manually in the tutorial PR (#157).